plugins/pulp_rpm/plugins/importers/yum/repomd/updateinfo.py

```diff
@@ -78,7 +78,11 @@ def process_package_element(element):
 
     issued_element = element.find('issued')
     if issued_element is not None:
-        package_info['issued'] = issued_element.attrib['date']
+        try:
+            package_info['issued'] = issued_element.attrib['date']
+        except KeyError:
+            # If there is a sub-element named 'date' instead of an attribute, use that
+            package_info['issued'] = issued_element.find('date').text
```

Comment:
What happens if ```date``` does not exist as a sub-element either?  Which exception is it going to bubble up?

Reply:
oh that's a good point. It probably needs to `raise` when there are 0 sub-elements also. A previous version I had did that. @dralley do you agree?

Reply:
That sounds reasonable